sdks/python/apache_beam/coders/stream.pxd

```diff
@@ -23,7 +23,7 @@ cdef class OutputStream(object):
   cdef size_t buffer_size
   cdef size_t pos
 
-  cpdef write(self, bytes b, bint nested=*)
+  cpdef write(self, b, bint nested=*)
```

Comment:
<!--new_thread; commit:57a73055b9447e7a1fe6ad674673254d9a614eeb; resolved:0-->
I'm not sure removing this type annotation is the right thing to do, for performance...

CC: @robertwb, who could provide more guidance.

Reply:
This wasn't meant to be permanent. I wanted to get some feedback on this in the pr reviews and then add a commit to fix it.